examples/create-react-app-with-typescript/tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": ".",
```

Comment:
Why is this needed?

Reply:
Without it there are a few warnings that flash on startup.

Reply:
What are the warnings?

Reply:
```
Found no baseUrl in tsconfig.json, not applying tsconfig-paths-webpack-plugin                              │                                                                                                           
Found no baseUrl in tsconfig.json, not applying tsconfig-paths-webpack-plugin
```

Reply:
Interesting, ok 🤷‍♂️